diracx-core/tests/test_secrets.py

```diff
@@ -15,8 +15,7 @@ def compare_keys(key1, key2):
 def test_token_signing_key(tmp_path):
     private_key = rsa.generate_private_key(
         public_exponent=65537,
-        # DANGER: 512-bits is a bad idea for prod but makes the test notably faster!
-        key_size=512,
+        key_size=1024,
```

Comment:
Can we avoid making the tests slower by using elliptic curves?

Reply:
In this PR? Can we leave for another one?